pkg/cmd/auth/login/login.go

```diff
@@ -138,6 +139,7 @@ func NewCmdLogin(f *cmdutil.Factory, runF func(*LoginOptions) error) *cobra.Comm
 	_ = cmd.Flags().MarkHidden("secure-storage")
 
 	cmd.Flags().BoolVar(&opts.InsecureStorage, "insecure-storage", false, "Save authentication credentials in plain text instead of credential store")
+	cmd.Flags().BoolVar(&opts.SkipSSHKeyPrompt, "skip-ssh-key", false, "Skip generate/upload SSH key prompt")
```

Comment:
No problem with this line; simply closest to command definition to suggest to update the `Long` usage description to call out that `gh auth login` actually does something around SSH keys aside from inferring it from the flag:

```go
                       Specifying %[1]sssh%[1]s for the git protocol will detect existing SSH keys to upload,
                       prompting to create and upload a new key if one is not found. This can be skipped with
                       %[1]s--skip-ssh-key%[1]s flag.
```

Reply:
Good point. Done.